### ISSUE TITLE
Doesn't give invite link

### ISSUE THREAD

**Issue by serverkur** [AnushK-Fro/Discord-Voice-Chat-Game-Activities#1]
Traceback (most recent call last):
  File "C:\Users\sunuc\Desktop\Discord-Voice-Chat-Game-Activities-main\Youtube\main.py", line 44, in <module>
    print("Generated Invite Link: https://discord.gg/" + response["code"]) # Print the invite link
TypeError: can only concatenate str (not "int") to str

**Comment by AnushK-Fro** [comment]
Is your token correct?

**Comment by serverkur** [comment]
yes

**Comment by nsde** [comment]
It's easy to fix! Just replace the `response["code"]` with `str(response["code"]` ;) Hope I could help.

**Comment by nsde** [comment]
Also, check out my "issue", I solved most of the problems! https://github.com/AnushK-Fro/Discord-Voice-Chat-Game-Activities/issues/2

**Comment by serverkur** [comment]
i found another code

**Comment by AnushK-Fro** [comment]
> It's easy to fix! Just replace the `response["code"]` with `str(response["code"]` ;) Hope I could help.

It should already be a string as long as it generated successfully.

**Comment by nsde** [comment]
> It should already be a string as long as it generated successfully.

Yeah, sorry I noticed that. So, can you please give me the error message or code (or both)?

**Comment by nsde** [comment]
Also - Make sure not to leak your credentials.
